Kentico/xperience-algolia
# Feature: App Setting to disable search indexing

### Motivation

Resolves #10 

### Checklist


### How to test

Set `AlgoliaSearchEnableIndexing` app setting to `false` and see no indexing happens.
